README.md

```diff
@@ -1,2 +1,22 @@
 # api-documentation-frontend
 Front-end repo for the API documentation site
+
+## Releasing to Production
+
+We use GitLab tags for deployment to Production. Follow these steps:
+
+1. **Ensure code is ready:** Merge all changes for the release into `main` and ensure they are tested.
+
+2. **Create a GitLab Release:** Go to "Releases" in [GitLab](https://gitlab.cee.redhat.com/insights-platform/api-documentation-frontend/-/releases), click "Create a new release". Add a versioned tag name (e.g., `v1.0.0`), a title, and any notes about the release.
```

Comment:
I think it will get difficult to decide what version to use. We should set some strategies or even use something automated to decide next version number (e.g. semantic/conventional commit)